pkgs/development/python-modules/sentencepiece/default.nix

```
@@ -0,0 +1,17 @@
+{ buildPythonPackage
+, stdenv
+, fetchgit
+, sentencepiece
+, pkgs
```

import on a per-packagke basis
```suggestion
, pkgconfig
```